.gitignore

```diff
@@ -6,7 +6,7 @@ dist/
 packages/dao-subgraph/src/helpers/ens.ts
 
 # Production subgraph
-types
+generated
```

Comment:
@ajsantander should we remove the generated file on the voting and token-manager specific subgraphs?

Reply:
I remove it